tests/nested/classic/hotplug/task.yaml

```
@@ -55,12 +55,7 @@ execute: |
     hotplug_add_dev1
 
     # sanity checks to make sure qemu setup is correct
-    for _ in $(seq 5); do
-        if nested_exec "udevadm info -e" | MATCH "ID_MODEL=QEMU_USB_SERIAL"; then
-            break
-        fi
-        sleep 1
-    done
+    nested_retry "--wait 1 -n 5 sh -c 'udevadm info -e | grep -qE \"ID_MODEL=QEMU_USB_SERIAL\"'"
```

On second though this should work too:
```suggestion
    nested_retry --wait 1 -n 5 sh -c 'udevadm info -e | grep -qE ID_MODEL=QEMU_USB_SERIAL'
```

---

done

---

If I remove the first and last " then I need to scape the '